### PR TITLE
add configure cache for the query

### PR DESCRIPTION
**Pull request by andrey1s**
- add cache interface 
- configure Lou cache by default
- add option set custom cache

### CODE REVIEW

File: handler/cache.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package handler
+
+// Cache for the graphqlHandler
+type Cache interface {
+	// Get looks up a key's value from the cache.
+	Get(key interface{}) (value interface{}, ok bool)
+	// Add adds a value to the cache.  Returns true if an eviction occurred.
+	Add(key, value interface{}) (evicted bool)
```
Comment (vektah) [inline handler/cache.go]: `evicted` is not used

Comment (andrey1s) [reply handler/cache.go]: its interface  for instead https://github.com/99designs/gqlgen/blob/master/handler/graphql.go#L384

Comment (vektah) [reply handler/cache.go]: It forces other implementations to return a value we don't use

**Comment by vektah** [comment]
Going to roll this into https://github.com/99designs/gqlgen/pull/885

**Pull request closed without merging**